### PR TITLE
test: move new test with flaky since it not work on current pipeline

### PR DESCRIPTION
**Pull request by evy-yang01**
add new test case into flaky

https://jira.tools.sap/browse/CXSPA-3399

### CODE REVIEW

File: projects/storefrontapp-e2e-cypress/cypress/e2e/regression/my-account/update-password.e2e-flaky.cy.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      before(() => {
+        standardUser.registrationData.email = generateMail(
+          randomString(),
+          true
+        );
+        cy.requireLoggedIn(standardUser);
+        cy.visit('/');
+      });
+
+      it('should display server error if agent try to modify customer password', () => {
```
Comment (i53577) [inline projects/storefrontapp-e2e-cypress/cypress/e2e/regression/my-account/update-password.e2e-flaky.cy.ts]: please add traceability to the test, append (CXSPA-XXXX) to the end of the description

**Review by niehuayang — DISMISSED** [review]
LGTM

**Comment by niehuayang** [comment]
Please double maintenance the bug fix to develop-6.4.x, https://github.com/SAP/spartacus/pull/17567 this PR is the same.

**Review by i53577 — APPROVED** [review]
LGTM

**Review by i53577 — APPROVED** [review]

File: projects/storefrontapp-e2e-cypress/cypress/e2e/regression/my-account/update-password.e2e-flaky.cy.ts (hunk 1 above)
Comment (evy-yang01) [reply projects/storefrontapp-e2e-cypress/cypress/e2e/regression/my-account/update-password.e2e-flaky.cy.ts]: aggree

**Review by niehuayang — APPROVED** [review]

**Review by Zeyber — APPROVED** [review]

**Pull request merged**